getting_started/introduction/learning_new_features.rst

```diff
@@ -121,10 +121,11 @@ information:
    Also, please don't take a picture with your phone, the low quality and screen
    reflections can make it hard to understand the image. Your operating system
    should have a built-in tool to take screenshots with the :kbd:`PrtSc` (Print
-   Screen) key.
+   Screen) key (macOS: use :kbd:`⌘ Command + ⇧ Shift + 3` for a full screen shot,
```

Comment:
```suggestion
   Screen) key (macOS: use :kbd:`cmd + Shift + 3` for a full screen shot,
```

Reply:
Now is it to be `Cmd` or `cmd`? Control is written capitalized as well.

Reply:
For now I've done them all as `Cmd` and `Opt`.